sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityLogOptions.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.identity.implementation;
+
+import com.azure.core.util.logging.LogLevel;
+
+public class IdentityLogOptions {
+    private LogLevel runtimeExceptionLogLevel;
+
+    public IdentityLogOptions() {
+        runtimeExceptionLogLevel = LogLevel.ERROR;
```

Comment:
Call the other ctor overload with `false` as the param to avoid duplication of default log levels.